.github/workflows/ci.yml

```diff
@@ -192,6 +205,7 @@ jobs:
           wget --retry-connrefused --waitretry=30 --read-timeout=30 --timeout=30 --tries=20 --no-clobber --quiet --directory-prefix=/tmp/cmake-download/ https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}-${OS}-x86_64.tar.gz
           tar -C /tmp/ -zxf /tmp/cmake-download/cmake-${CMAKE_VERSION}-${OS}-x86_64.tar.gz
           echo "/tmp/cmake-${CMAKE_VERSION}-${OS}-x86_64/bin" >> $GITHUB_PATH # Only available in subsequent steps!
+          cmake --version
```

Comment:
Just in case you're using this for diagnostics: This will never print the correct version, the `PATH` will not change in this step, so it will always print the globally installed version.

Same for the doxygen step, but here it would probably crash since doxygen is not installed by default.

Reply:
it prints a correct version for me (on linux)